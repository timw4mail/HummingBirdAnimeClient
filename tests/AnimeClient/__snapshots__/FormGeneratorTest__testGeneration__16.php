<?php declare(strict_types=1);

return '<select id="default_list" name="default_list">
    <option value="anime">Anime</option>
    <option value="manga">Manga</option>
</select>
';
