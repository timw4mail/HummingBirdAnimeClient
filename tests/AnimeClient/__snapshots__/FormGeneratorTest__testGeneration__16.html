<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<html><body>
<select id="default_anime_list_path" name="default_anime_list_path">
    <option value="watching">Watching</option>
    <option value="plan_to_watch">Plan to Watch</option>
    <option value="on_hold">On Hold</option>
    <option value="dropped">Dropped</option>
    <option value="completed">Completed</option>
    <option value="all">All</option>
</select>
</body></html>
